reproject/utils.py

```diff
@@ -43,3 +43,159 @@ def parse_output_projection(output_projection, shape_out=None):
             raise ValueError("Need to specify shape when specifying output_projection as WCS object")
 
     return wcs_out, shape_out
+
+def flatten(f):
```

Comment:
May be best to replace this with
https://github.com/aplpy/aplpy/blob/master/aplpy/regions.py#L177

(and this is a clear indication that we need `flatten_header` and `flatten_fits` as fits/wcs utils in astropy.core)